targets/mbedos5/source/jerry_port_mbed.c

```diff
@@ -90,10 +90,36 @@ jerry_port_get_time_zone (jerry_time_zone_t *tz_p) /**< timezone pointer */
 /**
  * Implementation of jerry_port_get_current_time.
  *
- * @return current timer's counter value in microseconds
+ * @return current timer's counter value in milliseconds
  */
 double
 jerry_port_get_current_time ()
 {
-  return (double) us_ticker_read ();
+  static uint32_t last_tick = 0;
+  static time_t last_time = 0;
+  static uint32_t skew = 0;
+
+  uint32_t curr_tick = us_ticker_read (); /* The value is in microseconds. */
+  time_t curr_time = time(NULL); /*  The value is in seconds. */
+  double result = curr_time * 1000;
+
+  /* The us_ticker_read () has an overflow for each UINT_MAX microseconds
+   * (~71 mins). For each overflow event the ticker-based clock is about 33
+   * milliseconds fast. Without a timer thread the milliseconds part of the
+   * time can be corrected if the difference of two get_current_time calls
+   * are within the mention 71 mins (note: because the comparision of two
+   * uint32 numbers, this interval has to be diveded by 2). Above that
+   * interval we can assume that the milliseconds part of the time is
+   * negligibe.
+   */
+  if (curr_time - last_time > 1800) {
+    skew = 0;
+  } else if (last_tick > curr_tick) {
+    skew = (skew + 33) % 1000;
+  }
+  result += (curr_tick / 1000 - skew) % 1000;
```

Comment:
There will be a clock skew in the millisecond part of time. Since the seconds are counted by the RTC, there will be no skew there.
In general we can assume that if two hours passed you are not interested in the correct milliseconds.